tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TypeSpec/TspInitTool.cs

```diff
@@ -31,9 +31,34 @@ ILogger<TypeSpecInitTool> logger
         private const string InitCommandName = "init";
 
         // command options
-        private readonly Option<string> outputDirectoryArg = new("--output-directory", "The output directory for the generated TypeSpec project. This directory must already exist and be empty.") { IsRequired = true };
-        private readonly Option<string> templateArg = new("--template", "The template to use for the TypeSpec project. Use azure-arm for resource management services, or azure-core for data plane services.") { IsRequired = true };
-        private readonly Option<string> serviceNamespaceArg = new("--service-namespace", "The namespace of the service you are creating. This should be in Pascal case and represent the service's namespace.") { IsRequired = true };
+        private readonly Option<string> outputDirectoryArg = new("--output-directory")
+        {
+            Description = "The output directory for the generated TypeSpec project. This directory must already exist and be empty.",
+            Required = true,
+        };
+
+        private readonly Option<string> templateArg = new("--template")
+        {
+            Description = "The template to use for the TypeSpec project. Use azure-arm for resource management services, or azure-core for data plane services.",
+            Required = true,
+        };
+
+        private readonly Option<string> serviceNamespaceArg = new("--service-namespace")
+        {
+            Description = "The namespace of the service you are creating. This should be in Pascal case and represent the service's namespace.",
+            Required = true,
+            Validators =
```

Comment:
~~Did you explicitly add this? This is literally the only time I've seen a `Validator` set up till now in this PR and I don't see why it's necessary for this guy.~~

Reply:
Moved from further down in the function.